src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -31,60 +63,53 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
-        {txHash ? (
-          <InlineEthHashInfo textSize="xl" hash={txHash} shortenHash={8} showCopyBtn explorerUrl={explorerUrl} />
-        ) : (
-          <Text size="xl" as="span">
-            {NOT_AVAILABLE}
-          </Text>
-        )}
-      </div>
-      {safeTxHash !== undefined && (
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            SafeTxHash:{' '}
-          </Text>
-          <InlineEthHashInfo textSize="xl" hash={safeTxHash} shortenHash={8} showCopyBtn />
-        </div>
-      )}
-      {nonce !== undefined && (
-        <div className="tx-nonce">
-          <Text size="xl" strong as="span">
-            Nonce:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {nonce}
-          </Text>
-        </div>
-      )}
-      {created && (
-        <div className="tx-created">
-          <Text size="xl" strong as="span">
-            Created:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {formatDateTime(created)}
-          </Text>
-        </div>
-      )}
-      <div className="tx-executed">
-        <Text size="xl" strong as="span">
-          Executed:{' '}
-        </Text>
-        <Text size="xl" as="span">
-          {executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE}
-        </Text>
-      </div>
       {txData?.operation === Operation.DELEGATE && (
         <div className="tx-operation">
           <DelegateCallWarning showWarning={!txData.trustedDelegateCallTarget} />
         </div>
       )}
-      {isMultiSendTxInfo(txInfo) && <TxInfoMultiSend txInfo={txInfo} />}
+      {isMultiSendTxInfo(txInfo) && (
+        <>
+          <TxInfoMultiSend txInfo={txInfo} />
+          <br />
+        </>
+      )}
+
+      {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
+      {safeTxHash && <TxDataRow title="SafeTxHash:" value={safeTxHash} inlineType="hash" hasExplorer={false} />}
+      {created && <TxDataRow title="Created:" value={formatDateTime(created)} />}
+      <TxDataRow title="Executed:" value={executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE} />
+
+      {/* Advanced TxData */}
+      {txData && (
+        <>
+          <StyledButtonLink onClick={toggleExpanded} color="primary" iconSize="sm" textSize="xl">
+            Advanced Details
+          </StyledButtonLink>
+          <CollapsibleSection show={expanded}>
+            {txData?.operation !== undefined && (
+              <TxDataRow
+                title="Operation:"
+                value={`${txData.operation} (${Operation[txData.operation].toLowerCase()})`}
+              />
+            )}
+            {safeTxGas && <TxDataRow title="safeTxGas:" value={safeTxGas} />}
+            {baseGas && <TxDataRow title="baseGas:" value={baseGas} />}
+            {gasPrice && <TxDataRow title="gasPrice:" value={gasPrice} />}
+            {gasToken && <TxDataRow title="gasToken:" value={gasToken} inlineType="hash" />}
```

Comment:
Can we move the null value check into the TxDataRow itself? This pattern is repeated many times.

Reply:
I will apply these changes

Reply:
Passed the null check to inside the <TxDataRow /> component